src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static final int SIZE_OF_ARRAY = 20;
```

Comment:
```suggestion
    public static final int MAX_SIZE_OF_ARRAY = 20;
```

it seems to me more informative that way